### PR TITLE
Update featuretools requirements

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Update featuretools version to >= 0.16.0
Update naming of cutoff time columns
Update notebook text to match results

### CODE REVIEW

File: PredictOlympicMedals.ipynb [hunk 1]
```diff
@@ -368,9 +368,9 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Built 295 features\n",
-      "Elapsed: 04:59 | Remaining: 00:00 | Progress: 100%|██████████| Calculated: 11/11 chunks\n",
-      "295 features generated\n"
+      "Built 371 features\n",
```
Comment (rwedge) [inline PredictOlympicMedals.ipynb:371]: We should update the cell above this to match the number of features created

File: PredictOlympicMedals.ipynb [hunk 2]
```diff
@@ -781,16 +773,16 @@
...
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "'Average AUC score is 0.94 with standard dev 0.032'"
+       "'Average AUC score is 0.949 with standard dev 0.024'"
```
Comment (rwedge) [inline PredictOlympicMedals.ipynb:782]: In the Analysis section below, we can updat the std numbers to match

Comment (thehomebrewnerd) [reply PredictOlympicMedals.ipynb:782]: Updated

**Review by rwedge — APPROVED** [review]
Looks good

**Pull request merged**